.github/workflows/ex_doc_version_check.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: ex_doc_version_check
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight
+
+
+jobs:
+  test:
+    name: Check ex_doc version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get current version
+        run: |
+            CURR=`curl -s https://api.github.com/repos/elixir-lang/ex_doc/tags | \
+                  grep "\"name\":" | \
+                  sort --reverse | \
+                  head -n 1 | \
+                  awk '{print substr($2, 2, length($2)-3)}'`
+            TITLE="\`ex_doc\` ${CURR} released"
+            ISSUE_EXISTS=`curl -s https://api.github.com/repos/starbelly/rebar3_ex_doc/issues | grep -c "\"title\": \"${TITLE}\""`
```

Comment:
This is also probably safe, if GitHub issues for `rebar3_ex_doc` don't "paginate" over a given limit, which is why I proposed the `EX_DOC_VSN` file.

Reply:
Absolutely. We can solve this by fixing GH API version via header. I can submit modifications next week

Reply:
> fixing GH API version via header

Does that prevent pagination? What I meant what that if you have too many issues, it's possible that the API won't give you all of them on the response and them you'll assume there's no issue (as per what you're searching) when there actually is one.

Reply:
I think yes. Currently docs dont mention any pagination, but we can try and fetch issues from some repo that have few k issues just to be sure. So current version of API should be supported and work consistently for a few years

Reply:
Yeah, doing this request on `erlang/otp`, as an example, brings up 30 results, while (atm) 275 are present...

Reply:
One idea might be to tag the issue and then retrieve it based on that? e.g. tag it with `_ ex_doc_version_check` and then retrieve it with `labels= ex_doc_version_check` (?)

Reply:
They'd have to abuse the label alongside the title, right? I'm Ok with your current solution, though.